modals/users.js

```diff
@@ -0,0 +1,30 @@
+const { v4 } = require('uuid');
+const md5 = require('md5');
+const { getUsers, setUsers } = require('./utils/storage.js');
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Install Prettier plugin for code formatting: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
I have prettier. I Was trying to avoid the future conflicts, after merging the branches.

Reply:
Don't worry about conflicts.

Commiting poorly formatted code is not a good idea.